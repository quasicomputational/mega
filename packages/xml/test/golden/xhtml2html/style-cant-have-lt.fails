<html xmlns="http://www.w3.org/1999/xhtml"><head><style>&lt;</style></head></html>
