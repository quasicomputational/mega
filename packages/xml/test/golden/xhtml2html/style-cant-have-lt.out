Error while HTMLifying: <style> can't have &lt; in it in the HTML syntax, but there's one at (1:58-1:60).